types/proxy-from-env/index.d.ts

```diff
@@ -10,4 +10,4 @@
  * @returns The URL of the proxy that should handle the request to the given
  *          URL.
  */
-export function getProxyForUrl(url: string): string;
+export function getProxyForUrl(url: string|URL): string;
```

Comment:
The URL object `getProxyForUrl` accept is [this](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/d129b9b777901659b1559f9355f9548496e72a2b/types/node/url.d.ts#L20), not the standard `URL`.

Reply:
I updated the types to use the `url` type, but the tests fail telling me the `"url"` module cannot be found. How can I fix this so the import works as expected? I checked other type definitions and they do this exactly the same as I do? (e.g. https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/parseurl/index.d.ts#L9 )